### ISSUE TITLE
Microsoft.Bcl.Async without workaround

### ISSUE THREAD

**Issue by WebDucer** [fsprojects/Paket#1062]
Hello.

Is there a solution for not working import of build targets of `Microsoft.Bcl.Build`?

We don't want to / can mix nuget and paket in our .Net 4.0 solution with over 100 projects (half of them use async/await with the library) like descriped in issue #1007.

**Comment by forki** [comment]
what's the concrete issue?

**Comment by WebDucer** [comment]
Like in #1007 I got the following error message during compilation:

` The name 'TaskEx' does not exist in the current context.`

I use the 2.0 version of paket.

**Comment by forki** [comment]
no I mean, what do you think should be changed in Paket?

**Comment by forki** [comment]
I still don't understand what we do wrong.

**Comment by WebDucer** [comment]
In #1007 at the end is a comment, that a new issue will be opend to solve this problem without the need of using nuget in parallel.

I don't found any and wanted to know, if a better way was found.

**Comment by forki** [comment]
;-)

I really want to help, but I don't know what's wrong. 

Is the targets file reference missing in the csproj after paket install? Or what is missing / going wrong?

**Comment by WebDucer** [comment]
Yes. The target is not in the project files.

**Comment by forki** [comment]
wow just found https://github.com/fsprojects/Paket/issues/618

**Comment by forki** [comment]
closing this - let's keep discussion in #1007
